fastlane/helper/plugin_scores_helper.rb

```diff
@@ -154,6 +169,12 @@ def append_git_data
                 tests_count += File.read(spec_file).scan(/ it /).count
               end
               self.data[:tests] = tests_count
+
+              actions = Dir["**/actions/*.rb"].map do |action_file|
+                FastlaneActionFileParser.new.parse_file(action_file)
+              end
+              # Result of the above is an array of arrays, this merges all of them into data[:actions]
+              self.data[:actions].concat(*actions)
```

Comment:
@milch, Any idea why I'm getting:

```
wrong number of arguments (given 8, expected 1)
...
bundler: failed to load command: fastlane (/Users/revolt/.gems/bin/fastlane)
ArgumentError: [!] wrong number of arguments (given 8, expected 1)
  /Users/revolt/Development/GitHub/revolter/fastlane/fastlane/helper/plugin_scores_helper.rb:177:in `concat'
  /Users/revolt/Development/GitHub/revolter/fastlane/fastlane/helper/plugin_scores_helper.rb:177:in `block (2 levels) in append_git_data'
  /Users/revolt/Development/GitHub/revolter/fastlane/fastlane/helper/plugin_scores_helper.rb:150:in `chdir'
...
```

when running `bundle exec fastlane update_docs`?

Reply:
It looks like it works with ruby 2.4 but not 2.3